### PR TITLE
adds infra menu PR for discussion per issue #333 & CPC conversation

### PR DESCRIPTION
**Pull request by jorydotcom**
This PR would add the foundation's "infrastructure menu of services" to the CPC repo. I was not 100% clear where it should ultimately live in the repo, but this PR is meant first to gather feedback and questions on the menu.

### CODE REVIEW

File: INFRASTRUCTURE_MENU.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## DNS
+The OpenJS Foundation will register and manage each project’s primary domain, and the .org, .dev, .net, and .com variants (as available).  The project’s primary domain must be transferred to the Foundation if the project desires the foundation to take financial and management responsibility for the domain.  Management of additional domain names may be requested, subject to budget approval.
+
+The OpenJS Foundation can either manage your DNS for you, or delegate to one of your own nameservers.  This is often required if you use a CDN (see below).
+
+## SSL Certificates
+The OpenJS Foundation will purchase 2-year wildcard SSL certs for each projects’ managed domains as needed.
+
+## Source Control
+By default all OpenJS Foundation projects have open source repositories in their own GitHub Organizations.  The OpenJS Foundation admin account must be added as administrator for each repository.  Two-factor authentication must be required for everyone in the organization.
```
Comment (tobie) [inline INFRASTRUCTURE_MENU.md]: > The OpenJS Foundation admin account must be added as administrator for each repository.

This bit also belong in the onboarding checklist, no?

File: INFRASTRUCTURE_MENU.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# OpenJS Foundation Project Infrastructure Services
+
+The OpenJS Foundation provides works with a number of services to provide critical infrastructure services to hosted projects.  We expect projects to be respectful of these services, to abide by their terms of use, and to use the for the good of the project and the OpenJS Foundation.
```
Comment (joesepi) [inline INFRASTRUCTURE_MENU.md]: > provides works with...

perhaps just provides?

Comment (joesepi) [reply INFRASTRUCTURE_MENU.md]: Also
>and to use ~the~ for the good

File: INFRASTRUCTURE_MENU.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## Servers
+Additional servers needed to run scripts, bots, or other project applications may also be deployed on Digital Ocean.  These resources are generously provided by Digital Ocean for free, so please be respectful of the arrangement and do not deploy overly intensive workloads such as CI/CD without discussion with the Foundation.  Additional servers can be procured on behalf of the project pending request and budget approval.
+
+## DNS
+The OpenJS Foundation will register and manage each project’s primary domain, and the .org, .dev, .net, and .com variants (as available).  The project’s primary domain must be transferred to the Foundation if the project desires the foundation to take financial and management responsibility for the domain.  Management of additional domain names may be requested, subject to budget approval.
+
+The OpenJS Foundation can either manage your DNS for you, or delegate to one of your own nameservers.  This is often required if you use a CDN (see below).
+
+## SSL Certificates
+The OpenJS Foundation will purchase 2-year wildcard SSL certs for each projects’ managed domains as needed.
```
Comment (joesepi) [inline INFRASTRUCTURE_MENU.md]: >projects’

project's

File: INFRASTRUCTURE_MENU.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+## Email
+The OpenJS Foundation uses Groups.io for mailing lists on the openjsf.org domain.  All projects are welcome to request their own lists on the @lists.openjsf.org subdomain.
+
+## Slack
+Projects are welcome to create channels on the OpenJS Foundation Slack (https://openjs-foundation.slack.com), or set up their own free Slack workspace.
+
+## Zoom
+Projects may request that standing meetings be added to the OpenJS Foundation calendar.  The OpenJS Foundation currently has two Zoom Pro meeting accounts, and one Zoom Webinar account which is capable of livestreaming.  Please be mindful of conflicts with other projects by requesting your meeting be scheduled on the shared calendar via email to operations@openjsf.org.   
+
+All OpenJS Foundation Zoom accounts have can host up to 300 participants, and meetings can be recorded as an .mp4 for posting to a project’s YouTube channel.
```
Comment (joesepi) [inline INFRASTRUCTURE_MENU.md]: >accounts ~have~ can host

File: INFRASTRUCTURE_MENU.md (hunk 1 above)
Comment (jorydotcom) [reply INFRASTRUCTURE_MENU.md]: indeed, good catch

Comment (jorydotcom) [reply INFRASTRUCTURE_MENU.md]: I captured this in #404; resolving here

**Comment by mhdawson** [comment]
I believe this can land based on the requirements in the governance: 

* there are no outstanding objections
* there are two approvals by CPC members
* the PR has been open for at least 72 hours

Landing

**Pull request merged**